replication-api-impl/src/main/java/com/connexta/replication/api/impl/data/ReplicatorConfigImpl.java

```diff
@@ -170,4 +164,61 @@ void setBidirectional(boolean bidirectional) {
   void setSuspended(boolean suspended) {
     this.suspended = suspended;
   }
+
+  @Override
+  public int hashCode() {
+    return Objects.hash(
+        super.hashCode(),
+        name,
+        source,
+        destination,
+        filter,
+        bidirectional,
+        description,
+        suspended,
+        lastMetadataModified);
+  }
+
+  @Override
+  public boolean equals(Object obj) {
```

Comment:
I only just realized Intellij can generate equals, hashcode, and toString for you...

Reply:
Yeah. I do prefer mine though :-)